operator/controller/src/main/java/io/apicurio/registry/operator/ApicurioDeployment.java

```diff
@@ -1,6 +1,6 @@
 package io.apicurio.registry.operator;
 
-import io.apicur.registry.v1.ApicurioRegistry;
+import io.apicurio.registry.operator.api.v3.ApicurioRegistry3;
```

Comment:
`ApicurioRegistry3` kind has version `v1`, even though it is intended for Registry `v3`. Either use `io.apicurio.registry.operator.api.v1` or what I did `io.apicurio.registry.operator.api.v3.v1`.

Reply:
:+1: